tools/medenv/med_environ.xml

```diff
@@ -0,0 +1,129 @@
+<tool id="iabiodiv_smartbiodiv_med_environ" name="Adds the environment variables to a dataset" version="0.0.1">
```

Comment:
please add a profile
```suggestion
<tool id="iabiodiv_smartbiodiv_med_environ" name="Adds the environment variables to a dataset" version="0.0.1" prfile="22.05">
```

Reply:
I added the profile version you suggested although did not review all the requirements that this profile make.